content/posts/Niladri_Phase_1_Report.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "CircuitVerse Vue Simulator: Phase 1 Report (GSoC 2024)"
+date: 2024-07-25T18:06:50+05:30
+author: Niladri Adhikary
+type: post
+---
+
+![gsoc-cover](/images/niladri-cover.png)
+
+It's been 8 weeks, and I have completed my first phase of GSoC. Throughout this phase, I have been working on the Vue Simulator for CircuitVerse. I learned a lot about Vue.js and TypeScript, and I have made significant progress in the project. I had to extend my phase 1 to 8 weeks due to exams, which gave me the time to complete this phase. During this time I really enjoyed working on the project and learned a lot of new things. I am looking forward to the next phase of the project.
```

Comment:
@niladrix719 Thanks for the blog, some common note:
- This is not a personal blog, it's a project blog which we are publishing as a group for the community and you are leading this.
- Prefer "we" sentence instead of "I".


Please, refactor the blog and ask for re-review